lib/puppetserver/ca/generate_action.rb

```diff
@@ -206,11 +217,38 @@ def sign_intermediate(ca_key, ca_cert, csr, valid_until, signing_digest)
           extension = ef.create_extension(*ext)
           cert.add_extension(extension)
         end
+
+        alt_names_ext = ef.create_extension("subjectAltName", subject_alt_names, false)
+        cert.add_extension(alt_names_ext)
+
         cert.sign(ca_key, signing_digest)
 
         cert
       end
 
+      def choose_alt_names(cli_alt_names, settings_alt_names)
+        if !cli_alt_names.empty?
```

Comment:
maybe ```.any``` would be easier to read?

Reply:
It's a string, not an array, does that work?

Reply:
:(

Reply:
Nothing else looked better.